app/views/teachers/mailer/magic_link.text.erb

```diff
@@ -1,7 +1,13 @@
-Hello <%= @resource.email %>
+# Confirm your email: apply for qualified teacher status (QTS)
```

Comment:
I thnk this is the subject line of the email. They live in a separate location: https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/config/locales/mailer.en.yml